tests/unit/backend/layers/api/test_portal_api.py

```diff
@@ -780,6 +822,130 @@ def test__get_all_collections_for_index(self):
         self.assertEqual(actual_collection["published_at"], collection.published_at.timestamp())
         self.assertEqual(actual_collection["revised_at"], collection.published_at.timestamp())
 
+        # test that the owner of a private collection can not see the private collection
+        headers = {"host": "localhost", "Content-Type": "application/json", "Cookie": self.get_cxguser_token()}
+        response = self.app.get(test_url.url, headers=headers)
+        self.assertEqual(200, response.status_code)
+        body = json.loads(response.data)
+
+        ids = [collection["id"] for collection in body]
+        self.assertIn(collection.collection_id.id, ids)
+        self.assertNotIn(private_collection.collection_id.id, ids)
+        self.assertNotIn(private_collection.version_id.id, ids)
+        self.assertNotIn(collection_to_tombstone.collection_id.id, ids)
+        self.assertNotIn(collection_to_tombstone.version_id.id, ids)
+
+        # test that super curators can not see the private collection
+        headers = {"host": "localhost", "Content-Type": "application/json", "Cookie": self.get_cxguser_token("super")}
+        response = self.app.get(test_url.url, headers=headers)
+        self.assertEqual(200, response.status_code)
+        body = json.loads(response.data)
+
+        ids = [collection["id"] for collection in body]
+        self.assertIn(collection.collection_id.id, ids)
+        self.assertNotIn(private_collection.collection_id.id, ids)
+        self.assertNotIn(private_collection.version_id.id, ids)
+        self.assertNotIn(collection_to_tombstone.collection_id.id, ids)
+        self.assertNotIn(collection_to_tombstone.version_id.id, ids)
```

Comment:
@NoopDog (regarding the behaviors you're testing here and above) this is already the behavior we expect: that users do not see their private Collections when on the `Collections` page; only when they are on the `My Collections` page do they see them. 

I guess we just lacked tests for this before?

Reply:
Correct.